cmd/server/server.go

```diff
@@ -0,0 +1,52 @@
+// Package server contains the code which can be used to start opi-spdk-bridge
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (c) 2022 Dell Inc, or its subsidiaries.
+// Copyright (C) 2023 Intel Corporation
+package server
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net"
+
+	"github.com/opiproject/opi-spdk-bridge/pkg/server"
+	"github.com/opiproject/opi-spdk-bridge/pkg/server/extension"
+
+	pb "github.com/opiproject/opi-api/storage/v1alpha1/gen/go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+)
+
+var (
+	port = flag.Int("port", 50051, "The Server port")
```

Comment:
A package should not unconditionally add flags to the global `flag.CommandLine`. Whoever builds the binary must be able to control the "API" towards the user.

One common pattern is to have a configuration struct with an associated `AddFlags` call. The binary then can reuse the common flags, but isn't forced to do it.

This also should be in a global variable.

Reply:
the same as above